soccer/src/soccer/strategy/agent/agent_action_client.cpp

```diff
@@ -170,13 +178,13 @@ void AgentActionClient::update_position(const rj_msgs::msg::PositionAssignment::
             next_position_ = std::make_unique<Goalie>(robot_id_);
             break;
         case 1:
-            next_position_ = std::make_unique<Defense>(robot_id_);
+            next_position_ = std::make_unique<Runner>(robot_id_);
             break;
         case 2:
-            next_position_ = std::make_unique<Offense>(robot_id_);
+            next_position_ = std::make_unique<Runner>(robot_id_);
             break;
         case 3:
-            next_position_ = std::make_unique<PenaltyPlayer>(robot_id_);
```

Comment:
this isn't quite right because these case #s need to correspond exactly with the enum defined in coach_node.hpp